File: directcall-java/app/src/main/java/com/sendbird/calls/quickstart/main/ApplicationInformationActivity.java
package com.sendbird.calls.quickstart.main;

import android.content.Context;
import android.os.Bundle;
import android.view.MenuItem;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import com.sendbird.calls.quickstart.R;
import com.sendbird.calls.quickstart.utils.PrefUtils;

public class ApplicationInformationActivity extends AppCompatActivity {

    private Context mContext;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_application_information);

        mContext = this;

        initViews();
    }

    private void initViews() {
        Toolbar toolbar = findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);
        if (getSupportActionBar() != null) {
            getSupportActionBar().setTitle(getString(R.string.calls_application_information));
            getSupportActionBar().setDisplayHomeAsUpEnabled(true);
            getSupportActionBar().setDisplayShowHomeEnabled(true);
            getSupportActionBar().setHomeAsUpIndicator(R.drawable.icon_arrow_left);
        }

        ((TextView)findViewById(R.id.text_view_application_id)).setText(PrefUtils.getAppId(mContext));
    }

    @Override
    public boolean onOptionsItemSelected(@NonNull MenuItem item) {
        if (item.getItemId() == android.R.id.home) {
            finish();
            return true;
        }
        return super.onOptionsItemSelected(item);
    }
}

File: directcall-kotlin/app/src/main/java/com/sendbird/calls/quickstart/main/ApplicationInformationActivity.kt
package com.sendbird.calls.quickstart.main

import android.os.Bundle
import android.view.MenuItem
import androidx.appcompat.app.AppCompatActivity
import com.sendbird.calls.quickstart.R
import com.sendbird.calls.quickstart.databinding.ActivityApplicationInformationBinding
import com.sendbird.calls.quickstart.utils.getAppId

class ApplicationInformationActivity : AppCompatActivity() {
    private lateinit var binding: ActivityApplicationInformationBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityApplicationInformationBinding.inflate(layoutInflater)
        setContentView(binding.root)
        initViews()
    }

    private fun initViews() {
        setSupportActionBar(binding.toolbar)
        supportActionBar?.let {
            it.title = getString(R.string.calls_application_information)
            it.setDisplayHomeAsUpEnabled(true)
            it.setDisplayShowHomeEnabled(true)
            it.setHomeAsUpIndicator(R.drawable.icon_arrow_left)
        }
        binding.textViewApplicationId.text = getAppId()
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        if (item.itemId == android.R.id.home) {
            finish()
            return true
        }
        return super.onOptionsItemSelected(item)
    }
}